packages/viewer/src/params/id.ts

```diff
@@ -0,0 +1,5 @@
+import type { ParamMatcher } from '@sveltejs/kit'
+
+export const match: ParamMatcher = param => {
+  return /^\w?(\d+)(.md)?$/.test(param)
```

Comment:
Is there a reason you want to allow things that _don't_ start with an expected letter prefix here?

Reply:
Ah, I got it ... because you want `spaces/1` to Just Work. I think that's fine in this case, but does complicate the case of `spaces/1/properties/1` if you want to rewrite it. In that case, you can't just rewrite the given id string, and instead have to deeply understand the position in the routes.

Reply:
I'm okay with requiring prefix; at least, supporting non-prefixed IDs when known in context can come later.